Library/docs/source/learn/physical_digital_realities/fiducial_markers.md

```diff
@@ -0,0 +1,249 @@
+
+* A marker placed on a table ensures digital objects remain fixed on the table's surface, even if the marker is shifted slightly.
+
+* Adjusting the marker's scale reflects changes in the perceived size of the marker, which in turn modifies the size and placement of AR objects.
+
+**Adjusting Digital Content Placement**
+
+Fiducial markers require their scale to match real-world dimensions. If a marker represents a 0.5-meter square, its scale property must reflect this size. This alignment ensures:
+
+* AR content appears proportional to the marker.  
+* The distance and positioning of other digital entities are consistent with real-world expectations.
+
+By managing these properties, you ensure that AR applications maintain spatial coherence across different devices and environments.
+
+### Inverse Transform and Space Rebasement
+
+The **inverse transform** of a fiducial marker is a mathematical transformation used to adjust the digital space origin relative to the marker. This process ensures that the marker becomes the reference point for all other digital entities.
+
+**Using Inverse Transforms**
+
+1. Calculate the inverse of the marker's transform, including position, rotation, and scale.  
+2. Use this inverse transform to rebase the digital space origin.  
+
+**Rebasing the Digital Space**
+
+Rebasing adjusts the digital space's origin to the fiducial marker's position. This allows other entities to implicitly inherit their relative placement based on the marker's new location. The result is a coordinated AR environment where:
+
+* Digital objects stay in the correct positions relative to the marker.  
+* Movement or rotation of the marker dynamically adjusts all related digital content.
+
+**Aligning Other Entities and Content**
+
+When rebasing the digital space, the marker's transform affects all associated entities. For example:
+
+* A fiducial marker scaled to 0.5 meters ensures digital objects placed at a distance of 1 meter appear twice as far from the marker as its width.  
+* A rotated marker realigns AR content, maintaining the correct orientation relative to the user's perspective.
+
+## Summary
+
+Fiducial markers are useful tools in CSP for aligning digital content with physical spaces. They provide a tangible and flexible anchoring solution, enabling accurate spatial positioning in augmented reality (AR) applications.
+
+By linking the physical and digital worlds, fiducial markers ensure AR content is displayed correctly, regardless of user perspective or device.
+
+1. **Fiducial Markers and AR Spatial Coherence**  
+   Fiducial markers create a consistent relationship between the physical environment and digital coordinate systems. Their transforms-comprising position, rotation, and scale-define how digital content aligns with the real world. Proper configuration ensures spatial coherence, making AR experiences intuitive and immersive.
```

Comment:
[nit] same comment regarding em-dashes

Reply:
Stripped out the pos,rot,scale part since the term transform implicitly includes it.

Reply:
Resolved in latest.